src/matrixroomhandler.ts

```diff
@@ -386,6 +386,139 @@ export class MatrixRoomHandler {
     return Promise.reject({err: "Unsupported", code: HTTP_UNSUPPORTED});
   }
 
+  public async HandleDiscordCommand(msg: Discord.Message) {
+    if (!msg.member.hasPermission("ADMINISTRATOR")) {
```

Comment:
This requirement seems a bit extreme, I'd check permissions *per command* since we might have commands that regulars might want.